rest-service/manager_rest/storage/management_models.py

```diff
@@ -474,6 +481,19 @@ def is_locked(self):
         return False
 
 
+class Token(SQLModelBase, CreatedAtMixin):
+    __tablename__ = 'tokens'
+
+    id = db.Column(db.String(10), primary_key=True, index=True)
+    secret = db.Column(db.String(255), nullable=False)
```

Comment:
string? maybe binary?

Reply:
if it's a string, i assume it's base64?

Reply:
It's the same as the password field on the users table.

Reply:
oh, right, it's a hash. Maybe `secret_hash`?

Reply:
Sure, why not